packages/context/src/context.ts

```diff
@@ -548,11 +554,10 @@ export class Context extends EventEmitter {
       });
       this.notificationQueue = undefined;
     }
-    if (this._parent && this._parentEventListeners) {
-      for (const [event, listener] of this._parentEventListeners) {
-        this._parent.removeListener(event, listener);
-      }
-      this._parentEventListeners = undefined;
+    if (this._parent && this.parentEventListener) {
+      this._parent.removeListener('bind', this.parentEventListener);
```

Comment:
The old code removes all the listeners, any reason why only moves the bind and unbind now?

Reply:
The old code uses a map for registered listeners. With this PR, only one function is used on bind/unbind events.